tools/perf/ResultsComparer/src/ResultsComparer/Program.cs

```diff
@@ -0,0 +1,62 @@
+﻿//---------------------------------------------------------------------
+// <copyright file="Program.cs" company="Microsoft">
+//      Copyright (C) Microsoft Corporation. All rights reserved. See License.txt in the project root for license information.
+// </copyright>
+//---------------------------------------------------------------------
+
+using System;
+using System.Globalization;
+using System.Threading;
+using CommandLine;
+using ResultsComparer.Core;
+using System.IO;
+using ResultsComparer.Core.Reporting;
+
+namespace ResultsComparer
+{
+    public class Program
+    {
+        public static void Main(string[] args)
+        {
+            // we print a lot of numbers here and we want to make it always in invariant way
+            Thread.CurrentThread.CurrentCulture = CultureInfo.InvariantCulture;
+            IResultsComparerProvider comparerProvider = ComparerProviderFactory.CreateDefaultProvider();
+            IReporterProvider reporterProvider = new ReporterProvider();
+
+            Parser.Default.ParseArguments<CommandLineOptions>(args)

```

Comment:
We could use the system.commandline package for this

Reply:
Maybe we can migrate to `System.CommandLine` package in a separate PR?

What benefits does it provide over this?

Reply:
It makes authoring console apps much easier by providing the necessary parsing and argument matching utilities.
it also generates a good help command which can be unbelievably valuable for this type of project.